### PR TITLE
[P4-2294][P4-2066] Show previous answer for a Person Escort Record question

### PR DESCRIPTION
**Pull request by teneightfive**
### What changed

This adds the ability to show a previous answer to a question on the Person Escort Record journey.

This is done as part of the rendering step within the framework step controller.

### Why did it change

In some cases users are moving the same person multiple times within a short period, for example a move to court each day of the week. In many cases the answers to the assessment remain the same. This feature allows users to see the previous answer to a question and review it rather than having to fill the information in again.

### Issue tracking

- [P4-2294](https://dsdmoj.atlassian.net/browse/P4-2294)
- [P4-2066](https://dsdmoj.atlassian.net/browse/P4-2066)

## Screenshots

<kbd>booksecuremove-pr-916 herokuapp com_move_347296fb-ef85-4cf7-a5e5-d19b642ae2ad_person-escort-record_risk-information_security</kbd>

#### Automated testing

- [x] Added unit tests to cover changes
- [ ] Added end-to-end tests to cover any journey changes

#### Manual testing

Has been tested in the following browsers:

- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Internet Explorer

### Environment variables

- [x] No environment variables were added or changed

### Other considerations

Commit messages with a `fix` or `feat` type are automatically used to generate the [changelog](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/CHANGELOG.md) and
[GitHub release notes](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/releases) during the release task. Please make sure they will read well on their own in a
summary of changes and that the commit body gives a more detailed description of those changes if necessary.

- [x] No new [Personally Identifiable Information (PII)](https://support.google.com/analytics/answer/6366371?hl=en) is being sent via analytics
- [x] Update [README](ministryofjustice/hmpps-book-secure-move-frontend/blob/master/README.md) with any new instructions or tasks

### CODE REVIEW

**Review by Andy-Millar — APPROVED** [review]
Great work. Pending copy to reflect answer has changed - all good. 

Question about prefilling medication to discuss - ie. dosage, last taken, next taken - concern with that information being highly likely to change and missed. I'd rather us not prefill this until we have the medication information segmented with 'add another' (as per previous design).

File: common/assets/scss/overrides/_govuk-frontend.scss [hunk 1]
```diff
@@ -32,6 +32,27 @@
   padding: govuk-spacing(1) govuk-spacing(2);
 }
 
+// Add support to show when a field contains a message
+.govuk-form-group--message:not(.govuk-form-group--error) {
+  padding-left: govuk-spacing(3);
+  border-left: 5px solid govuk-colour("blue");
+
+}
+.govuk-prefill-message {
```
Comment (solidgoldpig) [inline common/assets/scss/overrides/_govuk-frontend.scss:41]: Why not `.app-prefill-message`?

**Review by JadeNConroy — APPROVED** [review]

File: common/assets/scss/overrides/_govuk-frontend.scss (hunk 1 above)
Comment (teneightfive) [reply common/assets/scss/overrides/_govuk-frontend.scss:41]: I was actually a little torn about where to put this...let me have another stab at a name.

File: common/assets/scss/overrides/_govuk-frontend.scss [hunk 2]
```diff
@@ -32,6 +32,27 @@
   padding: govuk-spacing(1) govuk-spacing(2);
 }
 
+// Add support to show when a field contains a message
+.govuk-form-group--message:not(.govuk-form-group--error) {
```
Comment (solidgoldpig) [inline common/assets/scss/overrides/_govuk-frontend.scss:36]: Same question as the one below really...  Could it be done by adding an app-specific class instead

**Review by solidgoldpig — APPROVED** [review]

**Review by merlinc — APPROVED** [review]

**Pull request merged**